initializr-service/src/main/resources/application.yml

```diff
@@ -1252,6 +1252,33 @@ initializr:
           id: remote-shell
           description: CRaSH shell integration
           versionRange: "[1.0.0.RELEASE,1.5.0.RC1)"
+        - name: Codecentric Spring Boot Admin Server
+          description: Provides the Spring Boot Admin Server API (which can be paired with the UI)
+          id: spring-boot-admin-server
```

Comment:
Unfortunately, as [described in the guidelines for 3rd party starters](https://github.com/spring-io/initializr/wiki/Guidelines-for-3rd-Party-Starters), this naming is [not compliant](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-custom-starter-naming).